src/haxfred-slack-link-logging.js

```diff
@@ -1,47 +1,47 @@
-import { parseLink } from './link-helpers';
-import { post } from './benicio-helpers';
+import { parseLink } from './link-helpers'
+import { post } from './benicio-helpers'
 
 export default function haxfredSlackLinkLogging (haxfred) {
-  if (!haxfred.slack) return console.error('haxfred-slack-link-logging requires haxfred-slack'); // eslint-disable-line no-console
+  if (!haxfred.slack) return console.error('haxfred-slack-link-logging requires haxfred-slack') // eslint-disable-line no-console
```

Comment:
I believe the `// eslint-disable-line` is not necessary. I tried removing it and it passed.